### DISCUSSION TITLE
Is it possible to use this library today?

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [CommunityToolkit/Windows#166] [Q&A] [answered]
Even if no version has yet been released, is it possible to use this library in a project today?

**Comment by kmgallahan** [comment]
https://learn.microsoft.com/en-us/windows/communitytoolkit/nuget-packages

**Comment by niels9001** [comment, marked as answer]
> Even if no version has yet been released, is it possible to use this library in a project today?

Yes, you can! All the components have Nuget packages created - still in preview, though. You'd want to add the Toolkit preview Nuget feed and look for these as preview packages (see readme for details).

Once 8.0 is released, these will show up in the regular nuget feed.

Hope this helps!
